model-optimizer/extensions/front/tf/conv_ext.py

```diff
@@ -31,7 +31,8 @@ class Conv2DFrontExtractor(FrontExtractorOp):
     def extract(cls, node):
         attrs = tf_create_attrs(node, 2, 3)
         attrs.update({'op': __class__.op,
-                      'get_group': lambda node: 1,
+                      'get_group': lambda node: node.group if 'group' in node and node.group is not None else
+                      node.in_node(0).shape[node.channel_dims] / node.kernel_shape[node.input_feature_channel],
```

Comment:
We need layer tests for this change. Please, add new layers to cover this change tests and run existing one.